modules/nw-banp-audit-logging-concept.adoc

```diff
@@ -0,0 +1,65 @@
+// Module included in the following assemblies:
+//
+// * networking/openshift_network_security/AdminNetworkPolicy/logging-anp-policy.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="nw-banp-audit-logging-concept_{context}"]
+= BaselineAdminNetworkPolicy audit logging
+
+Audit logging is enabled in the `BaselineAdminNetworkPolicy` CR by annotating an ANP policy with the `k8s.ovn.org/acl-logging` key such as in the following example:
+
+.Example of annotation for `BaselineAdminNetworkPolicy` CR
+[%collapsible]
+====
+[source,yaml]
+----
+apiVersion: policy.networking.k8s.io/v1alpha1
+kind: BaselineAdminNetworkPolicy
+metadata:
+  annotations:
+    k8s.ovn.org/acl-logging: '{ "deny": "alert", "allow": "alert"}'
+  name: default
+spec:
+  subject:
+    namespaces:
+      matchLabels:
+          tenant: red
+  ingress:
+  - name: "default-allow-ns"
+    action: "Allow"
+    from:
+    - namespaces:
+          matchLabels:
+            tenant: blue
+  - name: "default-deny-ns"
+    action: "Deny"
+    from:
+    - namespaces:
+          matchLabels:
+            tenant: orange
+----
+====
+
+In the example, an event in which all of the namespaces with the label `tenant: blue` are allowed ingress to all of the namespaces with the label `tenant: red`, a log is generated.
```

Comment:
```suggestion
In the example, an event in which any of the namespaces with the label `tenant: blue` is allowed ingress to any of the namespaces with the label `tenant: red`, a log is generated.
```

BTW @asood-rh : I just noticed that the BANP+ANP samples provided here logically makes no sense. Making the BANP be the default deny {} to {} sounds better to me, what are your thoughts? there is no way ppl use the singleton BANP for a specific tenant case right? Can you please help with this part? Because otherwise, the ANP -> BANP transition here doesn't work together.

The other alternative is to give a preface to users that this is just to illustrate a logging example and functionally having "allow to blue" here makes no sense since nothing is actually blocking the "blue" traffic below that rule.

I might be overthinking this so feel free to correct me